docs/_user_guide/best_practices_screen.md

```diff
@@ -2,102 +2,62 @@
 
 Generally speaking, you can use the `screen` command, described in more detail below, to continue having your scripts run on the Yens after logging out, but a very important first question to ask is:
 
-<b>"Am I accessing anything stored in AFS (`/afs/***` or `/afs-home/***`) space?"</b>
+## "Am I accessing anything stored in AFS (`/afs/***` or `/afs-home/***`) space?"
```

Comment:
Let's get rid of this whole section -- the kerberos auth stuff is important (just like for oak) but i don't think it's worth going into on a page about screen